src/embed/sphere-renderer.js

```diff
@@ -148,7 +157,30 @@ SphereRenderer.prototype.createPhotosphere_ = function(texture, opt_params) {
     }
   }
 
-  var material = new THREE.MeshBasicMaterial({ map: texture });
+  var material;
+  if (texture.format === THREE.RGBAFormat && texture.flipY === false) {
```

Comment:
This should probably be fixed in THREE.js. I worry about this sort of code that is 1) fragile, and 2) likely to become deprecated as a dependency changes.